docker-app/qfieldcloud/core/exceptions.py

```diff
@@ -167,3 +167,7 @@ class ProjectAlreadyExistsError(QFieldCloudException):
     code = "project_already_exists"
     message = "This user already owns a project with the same name."
     status_code = status.HTTP_400_BAD_REQUEST
+
+
+class ReachedMaxOrgaMembers(Exception):
```

Comment:
```suggestion
class ReachedMaxOrgMembersError(ValidationError):
```

Reply:
So far I keep everything explicit without a lot of shortening. More typing, but clear explicitness.

Reply:
The thing I don't like is that it's not a validation error, and by the same token, should not to have to declare a `status_code`.

Reply:
ok, second thought. Define it like the rest of exceptions in the file, but extend `QFieldCloudException`.

YOLO if someone relies on `ValidationError`.

Reply:
Not sure if the just added status code is correct though.